### PR TITLE
Ms 1.3 updates michael

### PR DESCRIPTION
**Pull request by rabbitholiness**
Ready for initial review. Final images will probably need tweaking.

### CODE REVIEW

**Comment by GBKS** [comment]
Can you please remove all files and file changes that are unrelated to the content you're editing? That includes:
- .DS_Store
- Gemfile.lock
- LICENSE
- README.md
- assets/images/import-wallet/.DS_Store
- images/bitcoin-core-app-versions.png
- images/cross-platform-layouts.png

File: pages/milestones/1-3-import.md [hunk 1]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
 Below is an iteration of the wallet import flow, including migration and error paths. Many details need to be refined. Click to see it larger.
 
 {% include picture.html
-	image = "/assets/images/import-wallet/flow.png"
```
Comment (GBKS) [inline pages/milestones/1-3-import.md:29]: If you remove a reference to an image and the image is not used anymore, please remove the image file itself from the repo as well.

File: pages/milestones/1-3-import.md [hunk 2]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+	image = "/assets/images/import-wallet/import_flow.png"
 	retina = "/assets/images/import-wallet/flow@2x.png"
-	big = "/assets/images/import-wallet/flow-big.png"
+	big = "/assets/images/import-wallet/import_flow.png"
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]: Big size cannot be the same as the regular size. It should be high-res enough that someone can dig in and see all the details. For flows, that can end up being quite big.

File: pages/milestones/1-3-import.md [hunk 3]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
 Below is an iteration of the wallet import flow, including migration and error paths. Many details need to be refined. Click to see it larger.
 
 {% include picture.html
-	image = "/assets/images/import-wallet/flow.png"
+	image = "/assets/images/import-wallet/import_flow.png"
 	retina = "/assets/images/import-wallet/flow@2x.png"
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]: Make sure to also replace the retina image.

File: pages/milestones/1-3-import.md [hunk 4]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
 Below is an iteration of the wallet import flow, including migration and error paths. Many details need to be refined. Click to see it larger.
 
 {% include picture.html
-	image = "/assets/images/import-wallet/flow.png"
+	image = "/assets/images/import-wallet/import_flow.png"
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]: Regular image should be 800px wide. Retina twice that.

File: pages/milestones/1-3-import.md [hunk 5]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
 	retina = "/assets/images/import-wallet/flow@2x.png"
-	big = "/assets/images/import-wallet/flow-big.png"
+	big = "/assets/images/import-wallet/import_flow.png"
 	alt-text = "User flow mock-ups for importing an existing wallet."
 	width = 800
 	height = 368
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]: Make sure to update the height once you changed the images to the right dimensions.

File: pages/milestones/1-3-import.md [hunk 6]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+
+For those users who would like to use Bitcoin Core with a wallet, the app offers three main entry points for adding wallets:
+
+- Directly after the onboarding flow, if the user is doing a fresh install of the application.
+- Through the wallet selector dropdown, if he already has a different wallet. 
+- From the settings screen, if the application has been previously used in node only mode. 
+
+{% include picture.html
+	image = "/assets/images/import-wallet/entry-points.png"
+	retina = "/assets/images/import-wallet/entry-points@2x.png"
```
Comment (GBKS) [inline pages/milestones/1-3-import.md:51]: big image missing

File: pages/milestones/1-3-import.md [hunk 7]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+	alt-text = "Mockups showing the entry points to the wallet import flow."
+	width = 800
+	height = 368
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]: Make sure to update the height.

File: pages/milestones/1-3-import.md [hunk 8]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+
+{% include picture.html
+	image = "/assets/images/import-wallet/import-review-screen.png"
+	retina = "/assets/images/import-wallet/import-review-screen@2x.png"
+	alt-text = "Mockup showing the wallet review screen for a single-key and a multi-key wallet."
+	width = 400
+	height = 184
+%}
+
+The full details are available in the settings section for full review. Note that the information displayed this screen will vary based on the specific use case. For intance, if a user imports from an XPUB, the application will not now the wallet name.
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]:
```suggestion
The full details are available in the settings section for full review. The information displayed on this screen will vary based on the specific use case. For instance, if a user imports from an XPUB, the application will not know the wallet name.
```

File: pages/milestones/1-3-import.md [hunk 9]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+
+## Exception and error handling
+
+So far we discussed the happy path. However, there are various alternative scenarios to consider. These include:
+
+- The user tries to import a wallet file that has an outdated format.
+- The provided information (wallet file or pasted string) could not be processed.
+- Etc.
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]:
```suggestion
```

File: pages/milestones/1-3-import.md [hunk 10]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+
+{% include picture.html
+	image = "/assets/images/import-wallet/error-handling.png"
+	retina = "/assets/images/import-wallet/error-handling@2x.png"
+	alt-text = "Mockup showing a warning and an error message."
+	width = 400
+	height = 184
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]: big image missing and image size needs updated

File: pages/milestones/1-3-import.md [hunk 11]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+%}
+
+In all of these cases, we want to make sure that the application provides clear, concise and actionable feedback about:
+1. What went wrong (reason).
+2. What this means (impact).
+3. What does the user need to do (remedy).
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]:
```suggestion
3. What the user can do (remedy).
```

File: pages/milestones/1-3-import.md [hunk 12]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
 	height = 368
-%}
\ No newline at end of file
+%}
+
+## Entry points
+
+For those users who would like to use Bitcoin Core with a wallet, the app offers three main entry points for adding wallets:
+
+- Directly after the onboarding flow, if the user is doing a fresh install of the application.
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]:
```suggestion
- Directly at the end of the onboarding flow, if the user is doing a fresh install of the application.
```

File: pages/milestones/1-3-import.md [hunk 13]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+- Through the wallet selector dropdown, if he already has a different wallet.
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]:
```suggestion
- Through the wallet selector dropdown, if a wallet was already previously added. 
```

File: pages/milestones/1-3-import.md [hunk 14]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
\ No newline at end of file
+%}
+
+## Entry points
+
+For those users who would like to use Bitcoin Core with a wallet, the app offers three main entry points for adding wallets:
+
+- Directly after the onboarding flow, if the user is doing a fresh install of the application.
+- Through the wallet selector dropdown, if he already has a different wallet. 
+- From the settings screen, if the application has been previously used in node only mode.
```
Comment (GBKS) [inline pages/milestones/1-3-import.md:47]: Didn't we just have a small add wallet button in the top-left corner?

File: pages/milestones/1-3-import.md [hunk 15]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+
+
+## Import options
+
+Users have two main options to import a wallet: by loading a wallet file into the application or by copy-pasting an extended public key, bitcoin address or descriptor.
+
+{% include picture.html
+	image = "/assets/images/import-wallet/import-wallet-options.png"
+	retina = "/assets/images/import-wallet/import-wallet-options@2x.png"
+	alt-text = "Mockup showing the import screen."
```
Comment (GBKS) [inline pages/milestones/1-3-import.md:67]: big image missing and wrong image height.

File: pages/milestones/1-3-import.md [hunk 16]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+	width = 400
+	height = 184
+%}
+
+#### File upload
+
+Users can import a wallet by choosing a wallet file from their file system. In that case, the native file picker from their operating system. On desktop they can also just drag and drop the file into the application window.
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]:
```suggestion
Users can import a wallet by choosing a wallet file from their file system (via the native operating system file picker). On desktop, they can also drag and drop the file into the application window. 
```

File: pages/milestones/1-3-import.md [hunk 17]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+	retina = "/assets/images/import-wallet/entry-points@2x.png"
+	alt-text = "Mockups showing the entry points to the wallet import flow."
+	width = 800
+	height = 368
+%}
+
+
+## Import options
+
+Users have two main options to import a wallet: by loading a wallet file into the application or by copy-pasting an extended public key, bitcoin address or descriptor.
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]:
```suggestion
Users have two main options to import a wallet: by loading a wallet file into the application or by entering an extended public key, bitcoin address or descriptor.
```

File: pages/milestones/1-3-import.md [hunk 18]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+	alt-text = "Mockup showing the import screen."
+	width = 400
+	height = 184
+%}
+
+#### File upload
+
+Users can import a wallet by choosing a wallet file from their file system. In that case, the native file picker from their operating system. On desktop they can also just drag and drop the file into the application window. 
+
+#### Paste XPUB, bitcoin address or wallet descriptor
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]:
```suggestion
#### Copy & paste
```

File: pages/milestones/1-3-import.md [hunk 19]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+
+Users can also copy and paste an XPUB, bitocin address or wallet descriptor to import their wallet.
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]:
```suggestion
Users can also copy and paste an extended public key (XPUB), bitcoin address or wallet descriptor to import their wallet.
```

File: pages/milestones/1-3-import.md [hunk 20]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+
+
+## Wallet review screen
+
+After a wallet has been imported, the application displays the most important information and metadata about the wallet, so that the user can quickly assess whether everything worked as expected. The information displayed includes: 
+
+- Wallet name (if available)
+- Key scheme
+- Address type
+- Potentially more
```
Comment (GBKS) [inline pages/milestones/1-3-import.md]:
```suggestion
- Potentially more  based on wallet type
```

File: pages/milestones/1-3-import.md [hunk 21]
```diff
@@ -26,10 +30,94 @@ After the global navigation is established in milestone 1.1, we now focus on imp
...
+
+{% include picture.html
```
Comment (GBKS) [inline pages/milestones/1-3-import.md:90]: big image missing and image height incorrect.

**Review by GBKS — CHANGES_REQUESTED** [review]
Thanks for adding this. I'd like to request a whole bunch of changes. Many are just related to exporting and embedding images correctly, should be a piece of cake to take care of. There are also some text suggestions, and I left a separate comment about removing the files unrelated to your changes.

Could you please also use the same image background and spacing as the other pages? I always set them up with 50px padding, 50px gap between screens and #E5E5E5 as background color (here in [Figma](https://www.figma.com/file/ek8w3n3upbluw5UL2lGhRx/Bitcoin-Core-App-Design?type=design&node-id=7962%3A46301&mode=design&t=SyPxAlatweA5cHDu-1)). That will give us some nice consistency that allows readers to focus on the content itself.

**Comment by rabbitholiness** [comment]
@GBKS I think I addressed all of the issues you mentioned.

**Review by GBKS — CHANGES_REQUESTED** [review]
Thanks for fixing (almost) everything. Could you please remove the Gemfile and the 3 .DS_Store files from the PR? The .DS_Store files are likely hidden, so may need to enable showing hidden files in finder. Otherwise it's ready to go, IMHO.

**Comment by promag** [comment]
Add .gitignore to avoid committing `.DS_Store` and other unwanted files, see https://stackoverflow.com/a/107921.

**Comment by GBKS** [comment]
@promag good idea. I just did that on the `gh-pages` branch (we had a `.gitignore` already, I added a `.DS_Store` entry.

**Comment by rabbitholiness** [comment]
> Thanks for fixing (almost) everything. Could you please remove the Gemfile and the 3 .DS_Store files from the PR? The .DS_Store files are likely hidden, so may need to enable showing hidden files in finder. Otherwise it's ready to go, IMHO.

@GBKS I removed the Gemfile and the .gitignore (as suggested by promag), but I couldn't find any .DS_Store file, even after showing hidden files.

**Comment by GBKS** [comment]
ACK. Ready to go. I did 3 clean-up commits to:

- Restore the gitignore and Gemfile files (we do need them, just the version that is already in the repo, not the changes in this PR).
- Delete the _site directory. It contains the files your computer creates when running the site locally. We don't want them in the repo.
- Deleted .DS_Store files, which MacOS uses to store information about folders.

Much tidier now.

**Pull request merged**